templates/rack_mini_profiler.rb

```diff
@@ -0,0 +1,5 @@
+if ENV["RACK_MINI_PROFILER"].to_i > 0
```

Comment:
We've generally been moving towards using environment variables to configure changes between environments, vs configuring based on the environment name. We're experimenting with eliminating the `staging` Rails environment (as Heroku recommends), so this could be useful in that case.